### PR TITLE
Updated Android Template

### PR DESCRIPTION
**Pull request by d3fkn1ght**
Added SplashActivity.cs file and drawables.
Updated style.xml and .csproj
Removed tag from SplashActivity.cs

### CODE REVIEW

File: Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj [hunk 1]
```diff
@@ -101,6 +102,9 @@
       <Project>{A682884C-CB44-4A89-B752-27BB634E0ABE}</Project>
       <Name>BlankApp</Name>
     </ProjectReference>
+  </ItemGroup>
+    <ItemGroup>
+    <AndroidResource Include="Resources\drawable\splash_screen.xml" />
```
Comment (dansiegel) [inline Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj:107]: this being off on it's own wasn't intentional in the samples... you don't need the added ItemGroup... but move the AndroidResource next to the other AndroidResources in the project file

File: Xamarin.Forms/BlankApp/BlankApp.Android/SplashActivity.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    [Activity(Theme = "@style/MainTheme.Splash", 
+              MainLauncher = true, 
+              NoHistory = true)]
+    public class SplashActivity : AppCompatActivity
+    {
+        public override void OnCreate(Bundle savedInstanceState, PersistableBundle persistentState)
+        {
+            base.OnCreate(savedInstanceState, persistentState);
+            Log.Debug(TAG, "SplashActivity.OnCreate");
+        }
```
Comment (dansiegel) [inline Xamarin.Forms/BlankApp/BlankApp.Android/SplashActivity.cs:18]: this should be removed

File: Xamarin.Forms/BlankApp/BlankApp.Android/Resources/drawable/splash_screen.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:drawable="@color/launcher_background"/>
```
Comment (dansiegel) [inline Xamarin.Forms/BlankApp/BlankApp.Android/Resources/drawable/splash_screen.xml:3]: I noticed that the template currently doesn't have a colors.xml... grab the resource from the Samples template. You'll need to add the AndroidResource to the csproj. Also be sure to update the entire styles.xml from the Samples template

Comment (d3fkn1ght) [reply Xamarin.Forms/BlankApp/BlankApp.Android/Resources/drawable/splash_screen.xml:3]: I have copied the entire styles.xml over and updated the repo.
I also copied over the colors.xml file

File: Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj [hunk 4]
```diff
@@ -74,6 +74,7 @@
     <Compile Include="MainActivity.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <LinkDescription Include="LinkerSettings.xml" />
+	<Compile Include="SplashActivity.cs" />
```
[the remark this answers is not in the document]
Comment (d3fkn1ght) [reply Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj]: I have updated the indents

File: Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj (hunk 1 above)
Comment (d3fkn1ght) [reply Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj]: I have merged the Item Groups

File: Xamarin.Forms/BlankApp/BlankApp.Android/SplashActivity.cs (hunk 2 above)
Comment (d3fkn1ght) [reply Xamarin.Forms/BlankApp/BlankApp.Android/SplashActivity.cs]: I have removed the OnCreate method

**Comment by dansiegel** [comment]
This PR is out of date and the updates have been added.

**Pull request closed without merging**